frontend/src/components/core/Link.tsx

```diff
@@ -1,22 +1,29 @@
 import React from 'react';
 import { Link } from 'react-router-dom';
 import styled from 'styled-components';
-import { PrimaryButton } from './Button';
+import { PrimaryButton, SmallButton } from './Button';
 
 // Wrap a button inside of a Link to get the styling of a button
 const createButtonLink = (Button:any, props:any) => {
-  const { to, children, ...rest } = props;
+  const {
+    to,
+    style,
```

Comment:
nit: If possible could we keep most styling to the styled-components (for one off styles another option could also be just defining an XContent component at the top of the file)? Mainly cause it might be easier to forget about inline styles or have duplicate ones that all need to be changed for e.g.

Reply:
Yeah I think that's a good idea, I went a little heavy on the inline styling on this PR - I'll update this to use styled components.